ChameleonForms.Tests/FieldGenerator/DefaultFieldGeneratorTests.cs

```diff
@@ -180,55 +198,58 @@ public void Setup()
             return new DefaultFieldGenerator<TestFieldViewModel, T>(H, property, new DefaultFormTemplate());
         }
 
-        [Test]

```

Comment:
Pulled these tests into their own class - they were being sucked into a bunch of tests that overrode the base class and thus being re-run multiple times.